Resources/Prototypes/Entities/Objects/Specific/Research/anomaly.yml

```diff
@@ -42,10 +42,13 @@
           True: { visible: true }
           False: { visible: false }
   - type: PowerCellDraw
-    drawRate: 5
+    drawRate: 1
     useRate: 0
   - type: ProximityBeeper
     component: Anomaly
+    maximumDistance: 20
+    minBeepInterval: 0.15
+    maxBeepInterval: 1.00
```

Comment:
i feel like this sound is going to be incredibly spammy when you're next to the anomaly. Is there a reason why it was lowered?

Reply:
During testing I found that the rate that the beeping increase as you got closer to the anomaly didn't 'feel right'. I felt that once the anomaly is within view (about 10 tiles) the beeping should be getting fairly rapid, signifying its immenient discovery. 

With the orignal settings, at 10 tiles, the beep rate is every 0.8s. With the max interval reduced to 1, it's every 0.6s, and with the current settings, it's 0.5s. The beeping is quite rapid when standing directly ontop of the anomaly, but it's not overly irritating. And when standing just one tile away, it's a beep every 0.2s (vs 0.3s with the original settings). 

In anycase layers wouldn't be standing directly ontop of an anomaly very often, and will likely turning off the locator to save battery when they find the anomaly.

If you like, I can post a video to demonstrate.

Reply:
Here's a video demonstrating and tweaked range and beep intervals: https://youtu.be/jlcQ1yQd09k